src/Castle.Windsor/Windsor/WindsorContainer.cs

```diff
@@ -36,11 +36,11 @@ namespace Castle.Windsor
 	/// </summary>
 	[Serializable]
 	[DebuggerDisplay("{name,nq}")]
-#if (SILVERLIGHT)
-	public partial class WindsorContainer : IWindsorContainer
-#else
+#if FEATURE_REMOTING
 	[DebuggerTypeProxy(typeof(KernelDebuggerProxy))]
 	public partial class WindsorContainer : MarshalByRefObject, IWindsorContainer
```

Comment:
Could you put the `#if FEATURE_REMOTING` around just `MarshalByRefObject` like many other classes (i.e. add new lines) so the two definitions can't get out of sync.